Update/kage_001.txt

```diff
@@ -5377,7 +5377,7 @@ void main()
 			NULL, "\"...Don't look to me for agreement.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 38, "ps3/s08/38/320400032", 256, TRUE);
 	OutputLine(NULL, "告白されたばかりの女の子にそんな質問するなんて、あんたの発想の起源は美少女ゲームか？」",
-			NULL, " I don't want to ask a question like that to a girl who just received a confession. Do you think this is the introduction to a dating sim?\"", Line_Normal);
+			NULL, " Where'd you get the idea to ask a girl who just received a confession something like that, a dating sim?\"", Line_Normal);
```

Comment:
The sentence doesn't make sense to me (the ", a dating sim?" part  just seems appended to the sentence)

Reply:
Actually I get it, never mind :S